README.md

```diff
@@ -65,7 +65,7 @@ aa('setUserToken', 'USER_ID');
 | **`apiKey`**      | `string`       | None (required)          | The search API key of your Algolia application |
 | `userHasOptedOut` | `boolean`      | `false`                  | Whether to exclude users from analytics        |
 | `region`          | `'de' \| 'us'` | Automatic                | The DNS server to target                       |
-| `useCookie`       | `boolean`      | `true`                   | Whether to use cookie in browser environment. The anonymous user token will not be set if `false`. When `useCookie` is `false` and `setUserToken` is not called yet, sending events will throw errors because there is no user token to attach to the events. |
+| `useCookie`       | `boolean`      | `false`                  | Whether to use cookie in browser environment. The anonymous user token will not be set if `false`. When `useCookie` is `false` and `setUserToken` is not called yet, sending events will throw errors because there is no user token to attach to the events. |
```

Comment:
this warrants a bigger notice in the readme explaining how without setting useCookie users will be counted per visit

Reply:
I thought of adding it to the changelog. Do you think we need a notice on the readme?

Anyway, with `useCookie: false`, we do not generate anonymous user token at all. There is no token until explicitly set. So it's not counted per visit.

Reply:
yes, it should have a clear explanation in its own header IMO

Reply:
https://github.com/algolia/search-insights.js/pull/249/commits/e0e2e73e5b557e884a9f9eed4ba42417bf414cc9 wdty?